implementations/rust/ockam/ockam_vault/Cargo.toml

```diff
@@ -30,14 +30,13 @@ disable_default_noise_protocol = []
 OCKAM_XX_25519_AES256_GCM_SHA256 = []
 OCKAM_XX_25519_AES128_GCM_SHA256 = []
 OCKAM_XX_25519_ChaChaPolyBLAKE2s = []
+aws-lc = ["dep:aws-lc-rs"]
 
 # Feature (enabled by default): "std" enables functionality expected to
 # be available on a standard platform.
 std = [
   "ockam_macros/std",
   "ockam_node/std",
-  "aes-gcm/alloc",
```

Comment:
Could you please elaborate on this? Is there any reason we don't want `alloc` and `std` for `aes-gcm`?

Reply:
The code doesn't work without `alloc` so in the proposal i've moved it to the default feature. Then changed my mind to address this in the future with other `nostd` related changes, but forgot to restore it.
fixed

Reply:
Oh, the repetition part makes total sense